orvillebotmain.py

```diff
@@ -241,10 +280,61 @@ async def set_island_open(self, message: discord.Message, arg: str = ""):
 
 ####################### FUNCTIONS #######################
 
+def acnhsearch(searchterm):
+    search = searchterm.replace(" ", "+")
+    url = "https://villagerdb.com/search?game=nh&q=" + search
+    itemdata = {}
+    r = requests.get(url, verify=False)
```

Comment:
So my concern with these is that they seem to be blocking calls, curious if there's a way to do this asyncronously with event driven callbacks

Reply:
Suppose that's something to consider down the road anyway... We can just specify a timeout for now, let's put it at 2-3 seconds maybe?